### ISSUE TITLE
When do you think its good to let go of the organisation you're working with

### ISSUE THREAD

**Issue by sachindra149** [Neha/AMA#28]
1) When do you realise its time !!! While you're getting the tech stack to work on, you know the ambience you're working in, isn't that comfortable for you.

2) Looking for a change, what would you look out for? Better pay, better technology you would get to work on, better brand to be associated with. For instance, you work on ReactJS but you get to work on Javascript with an option to learn Python etc. With the change, you would atleast be comfortable with the new work environment.

**Comment by Neha** [comment]
@sachindra149  Thank you for asking.

1.  One thing I learned pretty late in my career was - nothing is as important as my mental peace and happiness. I worked for an organization where they always showed me how important I am for the organization and awesome projects I am working but I was into stress, no work-life-balance. This hit my health & when I asked for WLB I got negative feedback. I left that company at that moment. After that the moment I feel that I am not feeling happy to work in a company due to a team is not good, leadership is not good, etc. I just leave the company and move to a new job.

2. For me, the brand is not at all important. I would first look for the - is the company is providing a healthy environment - late-night sitting? Weekend working? WFH is available or not and so on. After that money, reason is we all are working for money. (If there is a company who is working on cutting edge tech but not paying then what is the point of working?). And at the last, the tech will come. 
Anyday, I can satisfy the developer inside me by doing side projects. 

Thank you,
Neha
